### PR TITLE
parallel output generation

### LINKED ISSUE #221: Parallelize generation of output files

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #221

### CODE REVIEW

**Comment by grzanka** [comment]
@nbassler what do you think about that ? On my Lenovo T440 with 4cores it means factor 3 speedup, on PLGRID cluster nodes up to 24cores it is even more.

It helps especially with slow matplotlib image generation.

File: pymchelper/run.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def add_default_options(parser):
     import pymchelper
     parser.add_argument("input", help='input filename, file list or pattern', type=str)
     parser.add_argument("output", help='output filename or directory', nargs='?')
+    parser.add_argument("--jobs", help='number of parallel jobs to use (-1 means all CPUs)', default=-1, type=int)
```
Comment (nbassler) [inline pymchelper/run.py]: can you and -j option?

Comment (grzanka) [inline pymchelper/run.py]: sure, added

Comment (grzanka) [inline pymchelper/run.py]: @nbassler is it better now ?

**Pull request merged**